checkov/sca_package/runner.py

```diff
@@ -35,8 +35,8 @@ def prepare_and_scan(
         runner_filter = runner_filter or RunnerFilter()
         excluded_file_names = excluded_file_names or set()
 
-        # skip complete run, if flag '--check' was used without a CVE check ID
-        if runner_filter.checks and all(not check.startswith("CKV_CVE") for check in runner_filter.checks):
+        # skip complete run, if flag '--check' was used without a CVE check ID or the license policies
+        if runner_filter.checks and all(not (check.startswith("CKV_CVE") or check.startswith("BC_CVE") or check.startswith("BC_LIC")) for check in runner_filter.checks):
```

Comment:
what do you say about moving this condition (which is not very short and is repeated) to a function, maybe in the file `checkov/common/sca/commons.py`